spec/vehicle_factory_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+RSpec.describe VehicleFactory do
+  before(:each) do
+    @factory = VehicleFactory.new
+    @wa_ev_registrations = DmvDataService.new.wa_ev_registrations
+  end
+
+  it 'exists' do
+    expect(@factory).to be_an_instance_of VehicleFactory
+  end
+
+  it 'creates a list of vehicles using an API' do
+    @factory.create_vehicles(@wa_ev_registrations)
```

Comment:
I don't think you need this test because it is essentially the same thing that you are testing on line 19